ADDON_HOOKS.md

```diff
@@ -255,7 +254,7 @@ Usually used to import assets into the application.
 ```js
 // https://github.com/yapplabs/ember-colpick/blob/master/index.js
 included: function colpick_included(app) {
-  this._super.included.apply(this, arguments);
+  this._super.included(app);
```

Comment:
Good catch. The source should probably get updated then: https://github.com/yapplabs/ember-colpick/blob/master/index.js#L10

I'll update the example here and submit a PR to ember-colpick.